<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detectionForm</class>
 <widget class="QWidget" name="detectionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detection</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : black; Color : rgb(152,243,6); border:1px solid white</string>
  </property>
  <widget class="QPushButton" name="previous">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>380</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : black; Color : rgb(152,243,6); border : 1px solid rgb(152,243,6)</string>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>380</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : black; Color : rgb(152,243,6); border : 1px solid rgb(152,243,6)</string>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load_btn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>380</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : black; Color : rgb(152,243,6); border : 1px solid rgb(152,243,6)</string>
   </property>
   <property name="text">
    <string>Load File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="home">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>380</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : black; Color : rgb(152,243,6); border : 1px solid rgb(152,243,6)</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QLabel" name="img_area">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>471</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : black; Color : rgb(152,243,6); border : 1px solid rgb(152,243,6)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>471</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white; border: 1px solid black</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>469</width>
      <height>269</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="status">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : black; Color : rgb(152,243,6); border : 0px</string>
   </property>
   <property name="text">
    <string>Detection</string>
   </property>
  </widget>
  <zorder>scrollArea</zorder>
  <zorder>img_area</zorder>
  <zorder>previous</zorder>
  <zorder>next</zorder>
  <zorder>load_btn</zorder>
  <zorder>home</zorder>
  <zorder>status</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
